azerothcore/azerothcore-wotlk
# fix(DB/Creature): Chillwind Chimaera falling through the floor

Fixed the spawn location of a Chillwind Chimaera that was falling through the floor when respawned.



## Changes Proposed:
-  slightly changes `position_z` of a Chillwind Chimaera (guid = 41253) 

## Issues Addressed:

- Closes #7406

## SOURCE:


## Tests Performed:

- Ran SQL, no errors
- Checked in-game
- The mob now does not fall through the floor (20+ respawns tested)


## How to Test the Changes:


1. `. go c 41253`
2. see that the Chillwind Chimaera is above ground
3. target the Chimaera and do `.die` and then `.respawn` and see that the Chimaera doesn't drop through the floor

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
